sites/CESNET-MCC.yaml

```diff
@@ -79,3 +79,6 @@ vos:
 - auth:
     project_id: 426b25102bdb49b8b6969cef5aabaecf
   name: fusion
+- auth:
+    project_id: 721251d671ea4268876043a9474ba3de
+  name: vip-competence-centre-biomed
```

Comment:
I missed the comment :)

Is `vip-competence-centre-biomed` a VO? Shouldn't this be `biomed`. If using some subgroup within biomed we need to check how to support that as we didn't do it before.

Reply:
Hi Enol,

Information that I got from Mirek that it is some subgroup inside biomed and at our cloud we created separate project for them.

Reply:
Ok this is not working so easily. Need to introduce some other changes in the cloud-info-provider to handle the auth so let's keep it as vip-competence-centre-biomed and I will work on fixing it